### ISSUE TITLE
Previous versions are not released to crates.io

### ISSUE THREAD

**Issue by subatiq** [EkardNT/bevy-tokio-tasks#24]
Versions 0.12.0 and 0.13.0 are not available on crates.io, yet README lists them as available.

```
error: failed to select a version for the requirement `bevy-tokio-tasks = "^0.12.0"`
candidate versions found which didn't match: 0.14.0, 0.11.0, 0.10.2, ...
location searched: crates.io index
```

**Comment by EkardNT** [comment]
Yes, in `d1d9dc162e8eba896d7e77fa4d63c11b88a6f39a` @foxzool was kind enough to contribute an update from Bevy 0.11 directly to 0.14, so 0.12 and 0.13 were indeed not actually published. Since there's no actual commit right with support for either of those versions, I'm going to go ahead and remove them from the Readme's documentation table.
